scripts/reinstall-rapiddisk-a.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# For Debian: v1.01
+# Warning: ECC Memory recommended.
+# Reinstall script Part 1/3 for Rapiddisk 8.2.0 Linux.
```

Comment:
I want to make this somewhat generic. If we can instead use "RapidDisk versions 8 and above"